src/inc/eventtracebase.h

```diff
@@ -426,7 +523,7 @@ namespace ETW
         
         static VOID DomainLoad(BaseDomain *pDomain, __in_opt LPWSTR wszFriendlyName = NULL)
         {
-            if (MICROSOFT_WINDOWS_DOTNETRUNTIME_PROVIDER_Context.IsEnabled)
+            if (ETW_PROVIDER_ENABLED(MICROSOFT_WINDOWS_DOTNETRUNTIME_PROVIDER))
```

Comment:
For now instead of littering !FEATURE_PAL in the codebase, I have defined it to be true for Non Windows. If a Logging system provides this functionality we could use it. LTTng does not provided this functionality, so we return true